plugins/GSdx/GS.cpp

```diff
@@ -45,6 +45,7 @@ static HRESULT s_hr = E_FAIL;
 #include "Window/GSWndEGL.h"
 
 extern bool RunLinuxDialog();
+extern bool RunwxDialog();
```

Comment:
You don't really need the extra extern here. You already extern it in the header, so you can just include the header and you'll have the function. I prefer it that way anyway.

Reply:
Taken care of, and I removed the commented old code while I was at it.